generate.py

```diff
@@ -42,6 +44,18 @@ def write_wav(waveform, sample_rate, filename):
     librosa.output.write_wav(filename, y, sample_rate)
     print('Updated wav file at {}'.format(filename))
 
+def create_seed(filename, sample_rate, quantization_channels, percent=50):
+    audio = librosa.load(filename, sr=sample_rate, mono=True)
+    audio = audio[0]
+
+    mu = quantization_channels - 1
```

Comment:
Can you re-use the `encode` TensorFlow function defined in `wavenet_ops.py`?
It might be more convenient to return the cut audio waveform here, and then run
`sess.run(encode)` in the if statement below.